quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMapper.java

```diff
@@ -346,24 +346,34 @@ public Builder<T> isEnabled(BooleanSupplier isEnabled) {
             return this;
         }
 
+        /**
+         * Set the validator, overwriting the current one.
+         */
         public Builder<T> validator(BiConsumer<PropertyMapper<T>, ConfigValue> validator) {
             this.validator = validator;
             return this;
         }
         
+        public Builder<T> appendValidator(BiConsumer<PropertyMapper<T>, ConfigValue> validator) {
+            var current = this.validator;
+            this.validator = (mapper, value) -> {
+                validator.accept(mapper, value);
+                current.accept(mapper, value);
```

Comment:
What would be the execution order? 

IMO, the validator execution order should be:

1. default validator or specified in `validator()`
2. top-down builder `appendValidator()`

So, If I'm not mistaken, this might be preferred, right?
```suggestion
                current.accept(mapper, value);
                validator.accept(mapper, value);
```

Reply:
This change was leftover from my initial resolution of the validation issues that cropped up, but is not longer needed given the separate methods. 

I can address this in a follow up, since it doesn't come into play with this PR. What it should do is collect the validation errors.

Reply:
Ok, agree to polish validations in a follow-up task. It's ok to provide these changes in this PR.